%YAML:1.0

#--------------------------------------------------------------------------------------------
# Camera Parameters. Adjust them!
#--------------------------------------------------------------------------------------------
CameraMat: !!opencv-matrix
  rows: 3
  cols: 3
  dt: d
  data:  [1.927941866078295e+03, 0., 9.595316580313911e+02,
           0.,  1.929172443177110e+03, 5.144334699977888e+02, 
           0.,  0.,                    1.]

DistCoeffs: !!opencv-matrix
  rows: 5
  cols: 1
  dt: d
  data: [-0.2272996302077433, 0.244566784018986, 4.524726839204985e-05, -0.0008645881261561915,  -0.]
  
ExtrinsicMat: !!opencv-matrix
  rows: 4
  cols: 4
  dt: d
  data: [0.01285381,-0.9999103, -0.00375912, 0.00470379, 
          -0.00171844, 0.00373733,-0.99999154,0.04719102, 
          0.9999159, 0.01286116, -0.00167,0.02269848, 
          0.0, 0.0, 0.0, 1.0]

Camera.width: 1920
Camera.height: 1080
